cluster-provision/k8s/check-cluster-up.sh

```diff
@@ -98,7 +98,10 @@ export KUBEVIRTCI_GOCLI_CONTAINER=quay.io/kubevirtci/gocli:latest
                 ${ksh} patch -n kubevirt kv kubevirt --type='merge' --patch '{"spec": {"configuration": {"seccompConfiguration": {"virtualMachineInstanceProfile": {"customProfile": {"localhostProfile" : "kubevirt/kubevirt.json"} } } } } }'
             fi
 
-            export SONOBUOY_EXTRA_ARGS="--plugin https://storage.googleapis.com/kubevirt-prow/devel/nightly/release/kubevirt/kubevirt/${LATEST}/conformance${arch_suffix}.yaml"
+            export LABEL_FILTER="((conformance) && !(RequiredSnapshotStorageClass,RequiredBlockStorage))"
+
+            export SONOBUOY_EXTRA_ARGS="--plugin https://storage.googleapis.com/kubevirt-prow/devel/nightly/release/kubevirt/kubevirt/${LATEST}/conformance${arch_suffix}.yaml --plugin-env kubevirt-conformance.E2E_LABEL=${LABEL_FILTER}"
```

Comment:
argument splitting is a pain. try dropping spaces?
`(conformance)&&(!RequiresSnapshotStorageClass)&&(!RequiresBlockStorage)`
(also note that logically the current version is not equivalent to the one I was advocating for)

Reply:
that looks better - thanks @akalenyu .